src/domain/Lists/DraggableList/DraggableList.examples.md

```diff
@@ -0,0 +1,22 @@
+#### Draggable List
+
+```jsx
+
+initialState = {
+    items: [
+    'lisa',
+    'needs',
+    'braces',
+    'dental plan!',
+  ]
+};
+
```

Comment:
I'd like for us to add a commented out hooks example here just so people aren't confused with how to use it. (Pretty sure styleguide doesnt support hooks in examples).

Something like
```
/*
  useState Hooks version

  const [items, setItems] = useState(['lisa', 'needs', 'braces', 'dental plan!'])

  <DraggableList setItems={setItems}>
    {items.map(item => (
        <div>
          {item}
        </div>
    ))}
  </DraggableList>
*/
```